app/bingey-api/controllers/watchlist.js

```diff
@@ -47,6 +47,24 @@ const addTitleToWatchlist = async (req, res) => {
   }
 };
 
+
+const deleteWatchlist = async (req, res) => {
+  try {
+    const watchlistId = req.params.id;
+    
+    Watchlist.deleteOne({_id: watchlistId}, (err) => {
+      if (err) {
+        res.status(204).json();
+        return;
+      }
+      res.json(watchlistId);
+    });
+
+  } catch (err) {
+    res.json({ errors: err });
```

Comment:
This returns a 200 response with `{"errors": "<Some error text>"}` if an exception is caught